efi-no-runtime/build.cmd

```diff
@@ -22,7 +22,7 @@
 @if "%1" == "clean" exit /B
 
 csc /nologo /debug:embedded /noconfig /nostdlib /runtimemetadataversion:v4.0.30319 zerosharp.cs /out:zerosharp.ilexe /langversion:latest /unsafe
-%ILCPATH%\ilc zerosharp.ilexe -o zerosharp.obj --systemmodule zerosharp --map zerosharp.map -O
+%ILCPATH%\ilc zerosharp.ilexe -o zerosharp.obj --systemmodule:zerosharp --map zerosharp.map -O
```

Comment:
Is this actually broken? I would prefer this to be consistent with `--map` parameter next to this.

IIRC the command line parser the compiler uses only mandates the colon when this is in a RSP file. It's a weird command line parser...

Reply:
Unfortunately yes. It is indeed broken. I better change --map to use colon (:) since it will be consistent. Maybe it can be practical to write tests on parser 😄

Reply:
It works without the colon on my box. If `--map` works without the colon, something might be off in your environment. I know the command line parser ILC currently uses is picky about the colon. AFAIK it's mandatory when the arguments are in a RSP file, but optional otherwise.

I would prefer to have these two consistent (either both with, or both without).

Reply:
It starts working without colons. I'm slightly puzzled. but at least I will get rid of .gitignore issues 😄